packages/legend-transac/src/@types/event/events.ts

```diff
@@ -56,6 +56,13 @@ export interface EventPayload {
         userId: string;
         userToUnblockId: string;
     };
+    /**
+     * Event to publish a product in the showcase microservice.
+     */
+
```

Comment:
The event is emitted by a particular microservice to the interested/subscribed microservices.

>"micro_who_emit:event_that_happen_in_the_micro"

In this case the event is emitted by "room-snapshot" and the event is "first_snapshot"

Remember alphabetical order also in the code